pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,64 +1,48 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, python2, gettext }:
-let
-  # pin requests version until next release.
-  # see: https://github.com/linkcheck/linkchecker/issues/76
-  python2Packages = (python2.override {
-    packageOverrides = self: super: {   
-      requests = super.requests.overridePythonAttrs(oldAttrs: rec {
-        version = "2.14.2";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0lyi82a0ijs1m7k9w1mqwbmq1qjsac35fazx7xqyh8ws76xanx52";
-        };
-      });
-    };
-  }).pkgs;
-in
+{ stdenv, lib, fetchFromGitHub, python2Packages, gettext }:
+
 python2Packages.buildPythonApplication rec {
   pname = "LinkChecker";
-  version = "9.3.1";
-
-  nativeBuildInputs = [ gettext ];
-  pythonPath = (with python2Packages; [
-    requests
-  ]) ++ [ gettext ];
-
-  checkInputs = with python2Packages; [ pytest ];
+  version = "9.4.0a";
```

Comment:
```suggestion
  version = "9.4.0";
```

404s

Reply:
Seems I accidentally bumped the version. I was doing that to force a rebuild to ensure my changes were okay. Strange that I didn't see the 404.

Reply:
Yep that can happen.

If you'd like to force a rebuild the best way would be to change `sha256` in the fetch function.
`lib` provides `lib.fakeSha256` for this purpose, so we can force a hash mismatch.

Reply:
Thanks for spotting that one BTW :grinning:

Reply:
Thanks again. I was looking for `nix-build --force` or something like that :smile:.